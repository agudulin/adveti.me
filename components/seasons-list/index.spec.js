import React from 'react'
import renderer from 'react-test-renderer'

import SeasonsList from './'

const basicProps = {}

describe('SeasonsList', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <SeasonsList {...basicProps} />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
