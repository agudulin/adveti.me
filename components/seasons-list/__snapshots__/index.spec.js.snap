// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeasonsList renders correctly 1`] = `
<ul
  className="seasons-list"
  data-jsx={3496861343}
>
  <li
    className="seasons-list__item"
    data-jsx={3496861343}
  >
    <a
      className="seasons-list__link"
      data-jsx={3496861343}
      href="/season?n=1"
      onClick={[Function]}
    >
      Season 
      1
    </a>
  </li>
  <li
    className="seasons-list__item"
    data-jsx={3496861343}
  >
    <a
      className="seasons-list__link"
      data-jsx={3496861343}
      href="/season?n=2"
      onClick={[Function]}
    >
      Season 
      2
    </a>
  </li>
  <li
    className="seasons-list__item"
    data-jsx={3496861343}
  >
    <a
      className="seasons-list__link"
      data-jsx={3496861343}
      href="/season?n=3"
      onClick={[Function]}
    >
      Season 
      3
    </a>
  </li>
  <li
    className="seasons-list__item"
    data-jsx={3496861343}
  >
    <a
      className="seasons-list__link"
      data-jsx={3496861343}
      href="/season?n=4"
      onClick={[Function]}
    >
      Season 
      4
    </a>
  </li>
  <li
    className="seasons-list__item"
    data-jsx={3496861343}
  >
    <a
      className="seasons-list__link"
      data-jsx={3496861343}
      href="/season?n=5"
      onClick={[Function]}
    >
      Season 
      5
    </a>
  </li>
  <li
    className="seasons-list__item"
    data-jsx={3496861343}
  >
    <a
      className="seasons-list__link"
      data-jsx={3496861343}
      href="/season?n=6"
      onClick={[Function]}
    >
      Season 
      6
    </a>
  </li>
  <li
    className="seasons-list__item"
    data-jsx={3496861343}
  >
    <a
      className="seasons-list__link"
      data-jsx={3496861343}
      href="/season?n=7"
      onClick={[Function]}
    >
      Season 
      7
    </a>
  </li>
  <li
    className="seasons-list__item"
    data-jsx={3496861343}
  >
    <a
      className="seasons-list__link"
      data-jsx={3496861343}
      href="/season?n=8"
      onClick={[Function]}
    >
      Season 
      8
    </a>
  </li>
</ul>
`;
